src/azure-cli/azure/cli/command_modules/iot/custom.py

```diff
@@ -449,6 +449,59 @@ def iot_hub_list(client, resource_group_name=None):
     return client.iot_hub_resource.list_by_resource_group(resource_group_name)
 
 
+def iot_hub_custom(instance,
```

Comment:
could the function's name be `update_iot_hub_custom`?

Reply:
Updated